demo/index.js

```diff
@@ -16,10 +18,60 @@ const camera = new THREE.PerspectiveCamera(
   1000
 );
 
-camera.position.set(0, 5, 10);
+camera.position.set(0, 0, 10);
 
 const controls = new OrbitControls(camera, renderer.domElement);
 
+const setupOptimizedScene = async (scene) => {
+  const duckLod = await getLod(2, "../lode-build/assets/Duck", "duck");
+  scene.add(duckLod);
+
+  const airplaneLod = await getLod(
+    2,
+    "../lode-build/assets/Airplane",
+    "airplane"
+  );
+  scene.add(airplaneLod);
+  airplaneLod.levels.forEach((level) => {
+    level.object.position.set(5, 0, 0);
+    level.object.scale.set(0.01, 0.01, 0.01);
+  });
+
+  const cameraLod = await getLod(2, "../lode-build/assets/Camera", "camera");
+  scene.add(cameraLod);
+  cameraLod.levels.forEach((level) => {
+    level.object.position.set(-5, 0, 0);
+    level.object.scale.set(1, 1, 1);
+  });
+
+  const dragonLod = await getLod(2, "../lode-build/assets/Dragon", "dragon");
+  scene.add(dragonLod);
+  dragonLod.levels.forEach((level) => {
+    level.object.position.set(0, -5, 0);
+    level.object.scale.set(0.5, 0.5, 0.5);
+  });
+};
+
+const setupNonOptimizedScene = async (scene) => {
+  const duckGltf = await loadGltfAsync("assets/Duck/duck.gltf");
+  scene.add(duckGltf.scene);
+
+  const airplaneGltf = await loadGltfAsync("assets/Airplane/Airplane.gltf");
+  scene.add(airplaneGltf.scene);
+  airplaneGltf.scene.position.set(5, 0, 0);
```

Comment:
Would it make sense to consolidate this logic in order to prevent inconsistencies in the setup for optimized and unoptimized version?